### ISSUE TITLE
react-native 0.62.0 - Animated: `useNativeDriver`

### ISSUE THREAD

**Issue by kperreau** [GeekyAnts/NativeBase#3109]
## I have gone through these following points

- [x] Check latest documentation: https://docs.nativebase.io/
- [x] Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [x] Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [x] Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- [] For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [] For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

#### node, npm, react-native, react and native-base version, expo version if used, xcode version
node: v12.14.1
npm: 6.14.4
react-native: 0.62.0
native-base: 2.13.12

#### Actual behaviour
Get some warning in react-native console since **react-native** `0.62.0`.
` WARN     Animated: "useNativeDriver" was not specified. This is a required option and must be explicitly set to "true" or "false"`.

#### Steps to reproduce
Just run a react-native app and use some components with animation without specify `useNativeDriver`, like `<Toast>`.

#### Is the bug present in both iOS and Android or in any one of them?
Both iOS and Android.

**Comment by Texicitys** [comment]
Same here :)

**Comment by sauravsid** [comment]
Same here. Please fix this issue

**Comment by astrahov** [comment]
Same `<Item>` with `floatingLabel `

```
<Form>
    <Item floatingLabel>
        <Label>Username</Label>
        <Input />
    </Item>
    <Item floatingLabel last>
        <Label>Password</Label>
        <Input />
    </Item>
</Form>
```

**Comment by mogol23** [comment]
for toast replace ~\node_modules\native-base\dist\src\basic\ToastContainer.js by this code https://pastebin.com/mDfeg13E
or you can add useNativeDriver manually, just search _animated.timing_ on nativebase module

sory for my bad english

**Comment by levibuzolic** [comment]
This would make a great first contribution for somebody to create a Pull Request for. It will mostly be a matter of finding all instances of `Animated.timing` or `Animated.spring` and adding `useNativeDriver: false`

Ideally `useNativeDriver: true` should be used for performance, but to match the current implementation of this library and avoid any headaches with working out which values can/cannot be natively driven and any elements that might have both native and JS driven animations it'll be safest just to throw `useNativeDriver: false` on all instances.

**Comment by AdeilsonESilva** [comment]
Same error here!

**Comment by sgonzalez98** [comment]
I have same error.

**Comment by dansokol** [comment]
Created a PR for this per @levibuzolic comments above: https://github.com/GeekyAnts/NativeBase/pull/3136

Currently using my fork to avoid the yellow warnings until they merge or fix and it's working fine for me now: 
"dependencies: {
...
"native-base": "https://github.com/dansokol/NativeBase.git",
...
}

**Comment by vladcast** [comment]
> Created a PR for this per @levibuzolic comments above: #3136
> 
> Currently using my fork to avoid the yellow warnings until they merge or fix and it's working fine for me now:
> "dependencies: {
> ...
> "native-base": "https://github.com/dansokol/NativeBase.git",
> ...
> }

Waiting for merge)

**Comment by zakaria-bali** [comment]
in ***\node_modules\native-base\dist\src\basic\Item.js search on 'animated.timing' and add  'useNativeDriver: false' to every callback function, this gonna hide the warning, (if u add useNativeDriver: true, you gonna face an error tell you that nativeDriver do not support Top (bla bala)), sorry for my bad english.

**Comment by Dheyson** [comment]
I got the warning using any of the input labeled from NativeBase. I saw that it was merged but still got the warning after the update. Does anyone know if it must make any configuration manually?

**Comment by dansokol** [comment]
I only updated the "src" directory.  

Wasn't sure if I should be committing to the "dist" directory as well.  Think it will be in the next update that compiles the src directory and updates the "dist" directory.

**Comment by DalerAsrorov** [comment]
Would be great to have the latest release soon

**Comment by asaadxheikh** [comment]
same yellow warning here :(

**Comment by AdrianoCahete** [comment]
Any update on this issue @akhil-geekyants ?

**Comment by cecodor** [comment]
Waiting for this

**Comment by rickeletro** [comment]
@astrahov 
> Same `<Item>` with `floatingLabel `
> 
> ```
> <Form>
>     <Item floatingLabel>
>         <Label>Username</Label>
>         <Input />
>     </Item>
>     <Item floatingLabel last>
>         <Label>Password</Label>
>         <Input />
>     </Item>
> </Form>
> ```

in ***\node_modules\native-base\dist\src\basic\Item.js search on 'animated.timing' and add 'useNativeDriver: false'  ex: _reactNative.Animated.timing(this.state.opacAnim,{toValue:0.7,duration:150, useNativeDriver: false}).start();}}

**Comment by n3rd253** [comment]
@levibuzolic it looks like there is a PR waiting for this to be rolled out.  Can we please get this merged?

**Comment by shivarajnaidu** [comment]
Guys.. this is impacting everyone using latest ReactNative .. it would be helpful to all if admin can rollout the fix soon ...

**Comment by djimenezlegend** [comment]
Go Team .. It impacting everyone using latest ReactNative ... Can you help us by making the latest release available please

**Comment by satsuya0114** [comment]
same warning... :(
don't want to downgrade rn version
plz fix it asap

**Comment by damasofc** [comment]
is there any quick solution for this?

**Comment by n3rd253** [comment]
@damasofc downgrade .net below 0.62


Or the admins could merge the PR and deploy @levibuzolic

**Comment by Alaa-Ben** [comment]
Hi ! any news on this ?

**Comment by Nemferno** [comment]
I've been using [patch-package](https://github.com/ds300/patch-package) to keep the changes until the PR merges. I highly recommend it as the tool keeps the changes whenever a new version is released and an issue was never reconciled in the new version. Nice band-aid until admins merge the PR.

**Comment by aravi365** [comment]
Any news on this?????????????

**Comment by aymenamaraa** [comment]
Same here, any news?

**Comment by mtsprojects** [comment]
Any update

**Comment by sktan810** [comment]
same here, any updates for this fix ?

**Comment by asim47** [comment]
for now on

> > Created a PR for this per @levibuzolic comments above: #3136
> > Currently using my fork to avoid the yellow warnings until they merge or fix and it's working fine for me now:
> > "dependencies: {
> > ...
> > "native-base": "https://github.com/dansokol/NativeBase.git",
> > ...
> > }
> 
> Waiting for merge)

Currently using this fork until the update rolls out.

**Comment by vuskhoza** [comment]
https://github.com/dansokol/NativeBase.git PR working for me as well

**Comment by adavecohen95** [comment]
Any luck on this? Any chance of getting a new npm version rolled out by this week?

**Comment by Nauglamir-dev** [comment]
I'm also waiting for a fix, any news ?

**Comment by dungpasoftware** [comment]
same warning -_- , I'm waiting updates for fix this

**Comment by vipulchasta** [comment]
Facing same issue from a long time and still no update from native-base developers yet.

We will have to wait till they officially fix it, or they do merge the existing PR for the fix.
Meanwhile use workaround to avoid irritating yellow warnings of useNativeDriver. 

Do addition of following lines in your App.js file.

```
import {YellowBox} from 'react-native';

componentDidMount() {
    YellowBox.ignoreWarnings(['Animated: `useNativeDriver`']);
}
```

**Comment by TamashiKaizen** [comment]
Hi, how can i fix this issue?

```
Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false` 
    in Item (at connectStyle.js:392)
```
```
            <Form>
                <Item floatingLabel>
                <Input placeholder="Username" />
                </Item>
            </Form>

```

**Comment by gutapuct** [comment]
When are they going to merge it into the release?

**Comment by kimkong88** [comment]
merge it please...

**Comment by Remigius2011** [comment]
Hi all,

Even if the urge to do so is very high, please refrain from posting "me too" comments.

The PR is available and merged (see message above from 8 May by @AdrianoCahete ), so it will eventually be released according to the release schedule. Until then, there is only a very slight annoyance, namely a yellow spot in development mode (it will not appear in production), but other than that everything works as designed. Further more, for those who absolutely need to remove this yellow spot from their development experience, there are various fixes posted in this Fred.

Cheers and have fun, Remi.

**Comment by DalerAsrorov** [comment]
@Remigius2011 it's been 3 months since this PR was merged and there has been no new releases for quite a while. If you are bothered by the "me too" comments maybe ask someone you know to release a new version that contains this fix.

**Comment by codler** [comment]
It is still an issue in React native 0.63.0

**Comment by YahiaBadr** [comment]
@Remigius2011 when is the expected time for the new release?

**Comment by codler** [comment]
I have [forked](https://github.com/codler/NativeBase) the repo to fix this issue.
```
npm i @codler/native-base
```

**Comment by darrylyoung** [comment]
Version `2.13.13` was just released and contains the fix for this issue.

**Comment by tonyea** [comment]
@darrylyoung thanks!! Tested and working.

**Comment by AdeilsonESilva** [comment]
Tks God, tks team!!! You are amazing!

**Comment by ewein** [comment]
I'm still receiving the issue after updating to 2.13.13 (also using Expo 38).

**Comment by ppulwey** [comment]
Still an issue
```
- node_modules/react-native/Libraries/Animated/src/AnimatedImplementation.js:365:6 in result.start
- node_modules/native-base/dist/src/basic/ToastContainer.js:1:3656 in onPanResponderRelease
```

**Comment by ImagineITSolutions** [comment]
Also have the issue with Toast. Same as above

**Comment by lvegh** [comment]
Recieving the following when closing toast:

Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false`

react-native: 0.62.2
react-base: 2.13.13

**Comment by nicolak** [comment]
> 
> 
> Recieving the following when closing manually toast:
> 
> Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false`
> 
> react-native: 0.62.2
> react-base: 2.13.13

I'm getting the same warning. 

react-native: 0.62.2
native-base: 2.13.13

**Comment by RikusWiehahn** [comment]
Also have the issue with Toast. Same as above
```
"native-base": "^2.13.13",
"react-native": "0.62.2",
```

**Comment by ilies-space** [comment]
```
  useEffect(() => {
    LogBox.ignoreLogs(['Animated: `useNativeDriver` was not specified.']);
    return () => {};
  }, []);
```

**Comment by cecodor** [comment]
Just getting this warning sometimes when tapping buttons making them unusable (version 2.13.13), restarting the app solves the issue

**Comment by Aung-Myint-Thein** [comment]
I found a very strange bug in Toast. When the toast is shown, the yellow warning also comes up for "useNativeDriver" although I am already on 2.13.13. After the toast is closed, I cannot click the buttons shown or covered by the toast area. Now, I replaced the Toast with RN Alert .. ugly but working now.

**Comment by djimenezlegend** [comment]
I have the same problem and resolved temporary with https://github.com/GeekyAnts/NativeBase/pull/3200, however in the last version 2.13.14 it was already fixed!

![image](https://user-images.githubusercontent.com/60449824/92488117-82e2b100-f1c4-11ea-9383-e18d5a0e49d6.png)

**Comment by marquina-abreu** [comment]
Upgrade to "native-base": "^2.13.14",  this solved the problem

**Comment by lewisjones28** [comment]
> Upgrade to "native-base": "^2.13.14", this solved the problem

Not for me it didn't...

**Comment by viniciuspsilvas** [comment]
> Upgrade to "native-base": "^2.13.14", this solved the problem
Not for me as well...

**Comment by Luan3642** [comment]
Add to your project the following import

import { YellowBox } from 'react-native';

YellowBox.ignoreWarnings([
  'Animated: `useNativeDriver` was not specified.',
]);

**Comment by BertoGz** [comment]
Hello Im also still facing issue with nativeDriver for the deck swiper.

**Comment by copleykj** [comment]
Just added PR #3278 to fix this issue in the DeckSwiper, Fab, and ToastContainer components.

**Comment by ticoprogrammer** [comment]
Is it already published on npm ? Im using 2.13.14 and im having that issue with FAB.

**Comment by MOHDSAMIR96** [comment]
> ## I have gone through these following points
> * [x]  Check latest documentation: https://docs.nativebase.io/
> * [x]  Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
> * [x]  Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
> * [x]  Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
> * [] For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
> * [] For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/
> 
> ## Issue Description
> #### node, npm, react-native, react and native-base version, expo version if used, xcode version
> node: v12.14.1
> npm: 6.14.4
> react-native: 0.62.0
> native-base: 2.13.12
> 
> #### Expected behaviour
> #### Actual behaviour
> Get some warning in react-native console since **react-native** `0.62.0`.
> ` WARN Animated: "useNativeDriver" was not specified. This is a required option and must be explicitly set to "true" or "false"`.
> 
> #### Steps to reproduce
> Just run a react-native app and use some components with animation without specify `useNativeDriver`, like `<Toast>`.
> 
> #### Is the bug present in both iOS and Android or in any one of them?
> Both iOS and Android.
> 
> #### Any other additional info which would help us debug the issue quicker.
WARN Animated: "useNativeDriver" was not specified. This is a required option and must be explicitly set to "true" or "false"`.
############## Solution ##############
->  where use Animated.timing fuction then add useNativeDriver = true  <-
like:  Animated.timing(this.animatedValue, {
        toValue: 0,
        duration: popupDuration,
        useNativeDriver: true
      }),

**Comment by MOHDSAMIR96** [comment]
WARN Animated: "useNativeDriver" was not specified. This is a required option and must be explicitly set to "true" or "false"`.
############## Solution ##############
-> where use Animated.timing fuction then add useNativeDriver = true <-
like: Animated.timing(this.animatedValue, {
toValue: 0,
duration: popupDuration,
useNativeDriver: true
}),

**Comment by lukewlms** [comment]
I don't repro as of 

    "native-base": "^2.15.2",

Release notes confirm this was an intended fix: https://github.com/GeekyAnts/NativeBase/releases/tag/v2.15.2

**Comment by copleykj** [comment]
Yes, this should be closed as it was fixed in #3278 and released in 2.15.2.
